test/e2e/v3/plugin_cluster_test.go

```diff
@@ -273,8 +273,8 @@ func curlMetrics(kbc *utils.TestContext) string {
 	cmdOpts := []string{
 		"run", "--generator=run-pod/v1", "curl", "--image=curlimages/curl:7.68.0", "--restart=OnFailure", "--",
 		"curl", "-v", "-k", "-H", fmt.Sprintf(`Authorization: Bearer %s`, token),
-		fmt.Sprintf("https://e2e-%v-controller-manager-metrics-service.e2e-%v-system.svc:8443/metrics",
-			kbc.TestSuffix, kbc.TestSuffix),
+		fmt.Sprintf("https://e2e-%[1]s-controller-manager-metrics-service.e2e-%[1]s-system.svc:8443/metrics",
```

Comment:
This shows that it not passing in the CI because of this change. 
I think it is out of the scope and if you just revert this one all will work as expected.

Reply:
This change is cosmetic. In any case, the namespace can be used directly as a subdomain of the SVC.